js/interface.js

```diff
@@ -527,6 +554,9 @@ function createDataSource(createOptions, options) {
       });
     }
 
+    $('[data-show-alive-source]').addClass('active-source');
```

Comment:
"Alive" as is not trashed? Please refer to them as "visible" or not using a prefix at all if possible.

Reply:
@squallstar fixed